ws/v1.md

```diff
@@ -0,0 +1,112 @@
+| 42        | Kovan        |
+| 3         | Ropsten      |
+| 4         | Rinkeby      |
+
+Filter parameters (optional): 
+*   `makerAssetType`: subscribes to new orders where the maker asset is of certain asset type (example: `ERC20`, `ERC721`)
+*   `takerAssetType`:  subscribes to new orders where the taker asset is of certain asset type (example: `ERC20`, `ERC721`)
+*   `makerAssetAddress`: subscribes to new orders where the contract address for the maker asset matches the value specified
+*   `takerAssetAddress`: subscribes to new orders where the contract address for the taker asset matches the value specified
+
+Order specific parameters (optional):
+*   `makerAssetData`: subscribes to new orders with the specified makerAssetData
+*   `takerAssetData`: subscribes to new orders with the specified takerAssetData
+*   `traderAssetData`: subscribes to new orders where either makerAssetData or takerAssetData has the value specified
+
+While there is some redundancy in supporting maker/takerAssetType, maker/takerAssetAddress, and maker/takerAssetData, they are actually all needed. For example, you cannot query "all ERC712 orders", or "all Cryptokitties orders", or "all ERC20 orders" with just maker/takerAssetData and need the other query params to do so.
+
+#### Response messages
+
+##### Update
+
+This message is sent whenever the relayer receives or hears about a new order, or when the `remainingFillableAmount` of an order changes because a fill occurred, or because the order was cancelled (in which case the remaining fillable amount would be 0).
+
+[See payload schema](https://github.com/0xProject/0x-monorepo/blob/development/packages/json-schemas/schemas/relayer_api_orderbook_channel_update_response_schema.ts#L1)
+
+```
+{
+    "type": "update",
+    "channel": "orders",
+    "requestId": "123e4567-e89b-12d3-a456-426655440000",
+    "payload":  {
+        "order": {
+            "makerAddress": "0x9e56625509c2f60af937f23b7b532600390e8c8b",
+            "takerAddress": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
+            "feeRecipientAddress": "0xb046140686d052fff581f63f8136cce132e857da",
+            "senderAddress": "0xa2b31dacf30a9c50ca473337c01d8a201ae33e32",
+            "makerAssetAmount": "10000000000000000",
+            "takerAssetAmount": "20000000000000000",
+            "makerFee": "100000000000000",
+            "takerFee": "200000000000000",
+            "expirationTimeSeconds": "42",
+            "salt": "67006738228878699843088602623665307406148487219438534730168799356281242528500",
+            "makerAssetData": "0x01323b5d4c32345ced77393b3530b1eed0f346429d",
+            "takerAssetData": "0x01ef7fff64389b814a946f3e92105513705ca6b990",
+            "exchangeAddress": "0x12459c951127e0c374ff9105dda097662a027093",
+            "signature": "0x012761a3ed31b43c8780e905a260a35faefcc527be7516aa11c0256729b5b351bc33"
+          },
+          "remainingFillableAmount": "5000000000000000"
+      }
+}
```

Comment:
It'll be a separate request for each order that changed? Maybe we could make this an array? Not sure about the efficiency gains to this, but it's highly likely that several orders get updated at once (e.g when a block gets mined), so might make sense.

Reply:
Can't hurt to make it an array